src/Pages/hokyunkim/Login/Login.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import "./Login.scss";
+import {withRouter} from 'react-router-dom'
+
+  class Login extends React.Component {
+    constructor() {
+      super();
+      this.state = {
+        idValue:'',
+        passValue:''
+      };
+    };
+
+    handleIdValue = (event) => {
+      console.log('onchange');
+      this.setState({
+        idValue:event.target.value
+      })
+    };
+
+    handlePassValue = (event) => {
+      this.setState({
+        passValue:event.target.value
+      })
+    };
+
+    goToMain = () =>{
+      this.props.history.push('/main-hokyun');
+    };
+
+    
+    render() {
+    
+        const{ idValue, passValue } = this.state;
+
+        return (
+      
+        <div className="loginHk">
+          <div className="loginPic">
+            <img alt="phone" src="/images/hokyunkim/app_phone.png" />
+          </div>
+          <div className="loginMain">
+            <div className="loginBody">
+              <div className="loginHeader">
+                <img alt="logo" src="/images/hokyunkim/logo_text.png" />
+              </div>
+              <div className="loginForm">
+                <div className="textId">
+                  <input  
+                  className= "loginIdInput" 
+                  onChange={this.handleIdValue} 
+                  type="text" 
+                  placeholder="전화번호, 사용자 이름 또는 이메일" />
+                </div>
+                <div className="pass">
+                  <input 
+                  className = "loginPassIn" 
+                  onChange={this.handlePassValue} 
+                  type="password" 
+                  placeholder="비밀번호" />
+                </div>
+                
+                  <button 
+                  className={
+                    idValue.includes('@') && 
+                    passValue.length > 5
+                      ? 'loginButton active' 
+                      : 'loginButton inactive'
```

Comment:
이 조건이 반복해서 쓰이니까 따로 변수로 빼서 쓸 수 있습니다
그리고 template literals 쓰면 더 줄일 수 있겠네요!
```suggestion
render(){
 return(
 const isValid =   idValue.includes('@') && passValue.length > 5;
 <button
  className={`loginButton ${isValid ? "active" : "inactive"}`}
```